src/transforms/index.js

```diff
@@ -39,9 +40,18 @@ const margin = directionFactory({
   prefix: 'margin',
 })
 const padding = directionFactory({ prefix: 'padding' })
-const fontVariant = tokenStream => ({
-  fontVariant: [tokenStream.expect(IDENT)],
-})
+const fontVariant = tokenStream => {
```

Comment:
Sorry to be nitpicky - can we put this in a separate file? This file was only really for very small definitions

Reply:
Yep, I agree I'll change this!